src/tests/topp/FileConverter_27_output.mzML

```diff
@@ -13,10 +13,10 @@
 			<cvParam cvRef="MS" accession="MS:1000294" name="mass spectrum" />
 		</fileContent>
 		<sourceFileList count="1">
-			<sourceFile id="sf_ru_0" name="FileConverter_27_input.mzML" location="file:///Users/alka/Documents/work/software/openms_dev/OpenMS/src/tests/topp">
+			<sourceFile id="sf_ru_0" name="FileConverter_27_input.mzML" location="file:///workspace/OpenMS/src/tests/topp">
 				<cvParam cvRef="MS" accession="MS:1000569" name="SHA-1" value="527bd77ecd57f783f88a772decb6b7ff6628a57f" />
 				<cvParam cvRef="MS" accession="MS:1000584" name="mzML format" />
-				<cvParam cvRef="MS" accession="MS:1000777" name="spectrum identifier nativeID format" />
+				<cvParam cvRef="MS" accession="MS:1000776" name="scan number only nativeID format" />
```

Comment:
this changed... probably need to double check

Reply:
https://github.com/OpenMS/OpenMS/blob/develop/src/tests/topp/FileConverter_27_input.mzML#L19

Reply:
Just found this on (under) my notifications pile. I think the change to `<cvParam cvRef="MS" accession="MS:1000776" name="scan number only nativeID format" />` better reflects how the id is used most times in the mzml / mzid code.
@jpfeuffer was there anything specific you wanted me to comment on?

Reply:
I think Timo tagged you. It is just about what you would suggest for files that have one or more input sources in the top-level of mzml but none of them is referenced in a spectrum.
Our current way was to only write sources that are referenced but this produces invalid mzmls since at least one source is necessary, while referencing is not.